rotkehlchen/api/rest.py

```diff
@@ -3984,23 +3986,30 @@ def get_user_added_assets(self, path: Optional[Path]) -> Response:
 
     @require_loggedin_user()
     def import_user_assets(self, path: Path) -> Response:
-        try:
-            import_assets_from_file(
-                path=path,
-                msg_aggregator=self.rotkehlchen.msg_aggregator,
-                db_handler=self.rotkehlchen.data.db,
-            )
-        except ValidationError as e:
-            return api_response(
-                result=wrap_in_fail_result(
-                    f'Provided file does not have the expected format. {str(e)}',
-                ),
-                status_code=HTTPStatus.CONFLICT,
-            )
-        except InputError as e:
-            return api_response(
-                result=wrap_in_fail_result(f'{str(e)}'),
-                status_code=HTTPStatus.CONFLICT,
-            )
+        zip_file = ZipFile(path)
+        with tempfile.TemporaryDirectory() as tempdir:
+            for file_name in zip_file.namelist():
```

Comment:
Why the for loop? Do you export multiple files?

Reply:
I export only one but this would be helpful in the case where a user could add multiple files in a zip file to import them

Reply:
Okay fair enough. Just asking in case I missed something.